presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -1055,6 +1068,12 @@ private Path getPartitionDirectory(String tableName, String partitionClause)
         return Paths.get(filePath).getParent();
     }
 
+
+    private Path getSinglePath(String tableName) {
```

Comment:
nit: getOnlyPath might be a better name - https://guava.dev/releases/20.0/api/docs/com/google/common/collect/Iterables.html#getOnlyElement-java.lang.Iterable-T-